src/features/bookings/components/TicketBody/Ean/Ean.tsx

```diff
@@ -0,0 +1,28 @@
+import { t } from '@lingui/macro'
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components/native'
+
+import { BookingOfferResponse } from 'api/gen'
+import { Typo, getSpacing } from 'ui/theme'
+
+type Props = {
+  offer: BookingOfferResponse
```

Comment:
ce composant a juste besoin d'un ISBN, je trouve ça dommage de lui passer tout l'`offer`

je propose
```suggestion
  isbn: string
```